app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,28 @@
+module CommentsHelper
+
+  def relative_time(creation_time)
+    diff = TimeDifference.between(creation_time, Time.now).in_general
```

Comment:
Rails/TimeZone: Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead. (https://github.com/bbatsov/rails-style-guide#time, http://danilenko.org/2012/7/6/rails_timezones)